composer.json

```diff
@@ -13,7 +13,7 @@
         "php": "^7.4 || ^8.0",
         "ext-hash": "*",
         "ext-json": "*",
-        "ext-mongodb": "^1.17.0",
+        "ext-mongodb": "^1.16.0",
```

Comment:
This was updated on-purpose by https://github.com/mongodb/mongo-php-library/commit/1acfab001810504214da8617e1500168086f7074

Reply:
The commit was there for initial testing and has already been removed locally